src/target_info.rs

```diff
@@ -0,0 +1,10 @@
+pub const RISCV_ARCH_MAPPING: &[(&str, &str)] = &[
+    ("riscv32gc", "riscv32"),
+    ("riscv32i", "riscv32"),
+    ("riscv32im", "riscv32"),
+    ("riscv32imac", "riscv32"),
+    ("riscv32imafc", "riscv32"),
+    ("riscv32imc", "riscv32"),
+    ("riscv64gc", "riscv64"),
+    ("riscv64imac", "riscv64"),
+];
```

Comment:
People often try to edit these kinds of files directly, so I think this file should come with a comment like
```rust
// This file is generated code. Please edit the generator
// in dev-tools/gen-target-info if you need to make changes.
```

Reply:
Added the prelude as a module document.